### PR TITLE
Style fix for Java discovery sample

### PR DESCRIPTION
**Pull request by saicheems**

### CODE REVIEW

File: src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_java/java_adexchangebuyer.v1.4.json.baseline [hunk 1]
```diff
@@ -389,7 +389,7 @@ public class AdExchangeBuyerExample {
     // The billing id associated with the budget being updated.
     long billingId = 0L;  // TODO: Update placeholder value.
 
-    // TODO: Assign values to desired fields of `requestBody`.
+    // TODO: Assign values to desired fields of `requestBody`:
     // Only assigned fields will be changed.
```
Comment (saicheems) [inline src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_java/java_adexchangebuyer.v1.4.json.baseline]: This seems a bit odd, @tcoffee-google what do you think?

Comment (tcoffee-google) [inline src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_java/java_adexchangebuyer.v1.4.json.baseline]: Just what I had in mind.

**Comment by tcoffee-google** [comment]
LGTM

**Comment by tcoffee-google** [comment]
LGTM

File: src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_java/java_adexchangebuyer.v1.4.json.baseline [hunk 2]
```diff
@@ -124,8 +124,9 @@ public class AdExchangeBuyerExample {
     // The account id
     int id = 0;  // TODO: Update placeholder value.
 
-    // TODO: Assign values to desired fields of `requestBody`.
-    // Only assigned fields will be changed.
+    // TODO: Assign values to desired fields of `requestBody`:
+    // TODO: Assign values to desired fields of `requestBody`. Only assigned
```
Comment (vchudnov-g) [inline src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_java/java_adexchangebuyer.v1.4.json.baseline]: Why is this repeated? DItto multiple places below.

Comment (saicheems) [inline src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_java/java_adexchangebuyer.v1.4.json.baseline]: Yeah I saw it, I'm fixing it

**Pull request merged**